### DISCUSSION TITLE
If this worth upgrade to `tiptap` when using `Prosemirror`?

### DISCUSSION THREAD

**Discussion by Shawnyeungcf** [ueberdosis/tiptap#2322] [General Questions]
Since I did lots of customise and custom `prosemirror` `node` on my current project. Wondering if this can be done 🤔

**Comment by hanspagel** [comment]
Sure, it’s possible. There’s nothing you can’t do with Tiptap that you can do with ProseMirror.

Tiptap comes with "more" than ProseMirror. If the "more" is worth switching depends on your project.
